math/src/main/java/com/powsybl/math/graph/UndirectedGraph.java

```diff
@@ -180,6 +181,42 @@ default int getVertexCapacity() {
      */
     int getEdgeVertex1(int e);
 
+    /**
+     * Return the edge objects connected to the specified vertex.
+     * This method throws a {@link com.powsybl.commons.PowsyblException} if the vertex doesn't exist.
+     *
+     * @param v the vertex index.
+     * @return an iterable of the edge objects
+     */
+    Iterable<E> getEdgeObjects(int v);
```

Comment:
For the 4 methods you added, the name is ambiguis, edges also have a "number" and also have an object associated like vertices. At first when just looking at the method name, without reading the javadoc, I misunderstand the purpose of the methods. 
Even if larger, maybe we could call it something like getEdgesConnectedToVertex, getEdgeObjectsConnectedToVertex

Reply:
Yes indeed, it's quite ambiguous, only the s of objects differentiates it from the `getEdgeObject(int e)`. Do you think the (old) existing `getEdgeObjects(int v1, int v2)` should also be renamed to something like `getEdgeObjectsBetweenVertices(int v1, int v2)` or `getEdgeObjectsConnectingVertices(int v1, int v2)`?